elements/pfe-clipboard/pfe-clipboard.ts

```diff
@@ -148,6 +155,11 @@ export class PfeClipboard extends LitElement {
     const useNewSuccessSlot = this.slots.hasSlotted('success') || !this.slots.hasSlotted('text--success');
     // TODO: Remove deprecated `text` slot and associated logic in 3.0
     const useNewLabelSlot = this.slots.hasSlotted('label') || !this.slots.hasSlotted('text');
+
+    if (!PfeClipboard.secure && this.copyType === 'navigator') {
+      return html`<!-- Browser supports navigator.clipboard API but current website is not behind [https] required by the specification. https://developer.mozilla.org/en-US/docs/Web/API/Clipboard -->`;
```

Comment:
🤔 not sure about this fallback. would prefer to fallback to old-school copy instead of requiring the user to opt-in to navigator.clipboard

Reply:
We can leave this part out, I actually had talked to @heyMP about if I should even add it.  I added it only because I was hiding the element completely, so it was just a hint if the dev didn't see the error and right clicked to inspect, definitely not married to the idea by any means.